requirements/base.txt

```diff
@@ -5,4 +5,5 @@ imagesize>=1.4,<1.5
 packaging>=20,<24
 pyyaml>=5.4,<7
 pywin32==306 ; sys_platform == 'win32'
+shellingham>=1.5.4,<1.6  ; sys_platform == 'linux'
```

Comment:
Also required for Mac?

Reply:
I am still wondering if shellingham should be used at all, a get_profile_file() doesn't use it, but this needs to be discussed.

Reply:
May we assume that linux rules to find profile file are also valid for Mac ?